README.md

```diff
@@ -18,13 +15,25 @@ The documentation might be found in the  [docs](./docs/README.asciidoc) director
 | [Keycloak](./deploy/crds/Keycloak.yaml)             | Manages, installs and configures Keycloak on the cluster |
 | [KeycloakRealm](./deploy/crds/KeycloakRealm.yaml)   | Represents a realm in a keycloak server                  |
 | [KeycloakClient](./deploy/crds/KeycloakClient.yaml) | Represents a client in a keycloak server                 |
+| [KeycloakBackup](./deploy/crds/KeycloakBackup.yaml) | Manage Keycloak database backups                         |
+
+## Deploying to a Cluster
+*Note*: You will need a running Kubernetes or OpenShift cluster to use the Operator
+
+1. Run `make cluster/prepare` # This will apply the necessary Custom Resource Definitions (CRDs) and RBAC rules to the clusters
+2. Run `kubectl apply -f deploy/operator.yaml` # This will start the operator in the current namespace
+
+### Creating Keycloak Instance
+Once the CRDs and RBAC rules are applied and the operator is running. Use the examples from the operator.
+
+1. Run `kubectl apply -f deploy/examples/keycloak/keycloak.yaml` 
```

Comment:
We should probably briefly mention other CRDs, like Clients, Users, Backups etc.